src/base/bittorrent/session.cpp

```diff
@@ -1259,6 +1261,10 @@ void Session::configure(libtorrent::settings_pack &settingsPack)
     if (isDHTEnabled())
         settingsPack.set_str(libt::settings_pack::dht_bootstrap_nodes, "dht.libtorrent.org:25401,router.bittorrent.com:6881,router.utorrent.com:6881,dht.transmissionbt.com:6881,dht.aelitis.com:6881");
     settingsPack.set_bool(libt::settings_pack::enable_lsd, isLSDEnabled());
+
+    settingsPack.set_int(libt::settings_pack::choking_algorithm,
+        chokingAlgorithm() ? libt::choking_algorithm_t::rate_based_choker : libt::choking_algorithm_t::fixed_slots_choker);
+    settingsPack.set_int(libt::settings_pack::seed_choking_algorithm, seedChokingAlgorithm());
```

Comment:
`seed_choking_algorithm` is being set in the session constructor so you need to remove it from there too.